AuroraEditor/Base/MenuBarItem/StatusItem.swift

```diff
@@ -10,6 +10,13 @@ import Foundation
 import SwiftUI
 
 extension AppDelegate {
+    private var appVersion: String {
+        Bundle.versionString ?? "No Version"
```

Comment:
We use return statements everywhere.

```suggestion
        return Bundle.versionString ?? "No Version"
```